DuckDuckGo/BookmarksViewController.swift

```diff
@@ -897,7 +897,11 @@ class BookmarksViewController: UIViewController, UITableViewDelegate {
             }
 
             syncPromoViewHostingController.view.translatesAutoresizingMaskIntoConstraints = false
-            addChild(syncPromoViewHostingController)
+            // This is needed to ensure the toolbar displays correctly on iOS 15
+            if #available(iOS 16.0, *) {
+                addChild(syncPromoViewHostingController)
+                syncPromoViewHostingController.didMove(toParent: self)
```

Comment:
That's a minor, since we're not doing any transition here, but this should be called after `headerView.addSubview(syncPromoViewHostingController.view)` (i.e. transition is complete).